src/azure-cli/HISTORY.rst

```diff
@@ -15,6 +15,10 @@ Release History
 
 * Remove support for Python 3.4
 
+**HDInsight**
+
+* Support for creating a Kafka cluster with Kafka Rest Proxy
```

Comment:
Declare your package version update here. e.g.
` Upgrade azure-mgmt-hdinsight to 1.3.0`

Reply:
> Declare your package version update here. e.g.
> ` Upgrade azure-mgmt-hdinsight to 1.3.0`

Added